test/AllInOne.Formatted.cs

```diff
@@ -948,7 +948,6 @@ class CSharp6Features
 
         private void NoOp()
         {
-            var foo = (2, MethodCallExpression());
```

Comment:
As mentioned on another commit, just rebase your code and simply do not introduce this line.